test-requirements.txt

```diff
@@ -1,3 +1,3 @@
 PyVirtualDisplay==0.1.5
-selenium==2.49.0
+selenium==2.49.2
```

Comment:
Also we're running `2.49.0`, so I don't know why we would be encountering "No JSON object could be decoded" already if it was introduced in `2.49.0`... very odd.